v2/cn/ann-1019009001.md.tt2

```diff
@@ -20,6 +20,9 @@ on this [Download page](download.html).
 * Based on a very recent mainline [NGINX](nginx.html) core [% major_ver %].
 * Many fixes imported from Mike Pall's upstream [LuaJIT](https://github.com/openresty/luajit2#readme) repository.
 * Introduce a new macro `LUAJIT_TEST_FIXED_ORDER` for fixed-order traversal of lua tables.
+It can be turned on by appending `LUAJIT_TEST_FIXED_ORDER=1` to `--with-luajit-xcflags`
+when building openresty by running `./configure` script. This feature is disabled by default,
```

Comment:
This option is disabled by default. It can be enabled by passing the `--with-luajit-xcflags LUAJIT_TEST_FIXED_ORDER=1` option to the `./configure` command line while building OpenResty from source.